lib/decimal/decimal.h

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) 2018-2019, NVIDIA CORPORATION.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// C and C++ API for binary-to/from-decimal conversion package.
+
+#ifndef FORTRAN_DECIMAL_H_
+#define FORTRAN_DECIMAL_H_
```

Comment:
I think the include guard should be `FORTRAN_DECIMAL_DECIMAL_H_`.

Reply:
Yes, of course.